README.md

```diff
@@ -121,6 +121,10 @@ func main() {
 }
 ```
 
+## Ports
+
+- [nodesplash](https://github.com/codetheweb/nodesplash): a TypeScript / Node.js port by [codetheweb](https://github.com/codetheweb)
+
```

Comment:
Haha nice :+1: Do you mind moving this below `## Quick links`? Since these are meant to be quickly accessed :cat:

Reply:
Updated 👍